Update IFCTunnelDep3_Sys.ifc

AdamMackellar12d opened the pull request:
Continuation of PR #174
Updated IFCTunnelDep3_Sys.ifc to fix some of the issues in mentioned in PR#174

AdamMackellar12d committed:
Update IFCTunnelDep3_Sys.ifc

Updated file to fix some of the requested changes

AdamMackellar12d committed:
Update IFCTunnelDep3_Sys.ifc

Fix up IfcQuantity measure types
Add UsageType and PredefinedType to IfcTunnelPart
Connect IfcAlignment directly to IfcProject
Set measure type to IfcVolumetricFlowRateMeasure for IfcPropertyBoundedValue
Clean up spatial structure heirarchy

AdamMackellar12d committed:
Update IFCTunnelDep3_Sys.ifc

Use only 1 IfcSite

AdamMackellar12d committed:
Update IFCTunnelDep3_Sys.ifc

Move water elements into IfcTunnel spatial structure

AdamMackellar12d committed:
Create IFCTunnelDep3_Geo.ifc

First draft of Sprint3_Geo

SergejMuhic commented:
@AdamMackellar12d 
#62 = IFCDIRECTION((-0., 0., 0.));

This one produces a problem.

AdamMackellar12d committed:
Update IFCTunnelDep3_Geo.ifc

Remove IFCDIRECTION with 0 magnitude

AdamMackellar12d committed:
Update IFCTunnelDep3_Geo.ifc

Add in upto sprint 2.3 requirements

AdamMackellar12d committed:
Update IFCTunnelDep3_Geo.ifc

Fix Root spatial structure placementRelTo

AdamMackellar12d committed:
Update IFCTunnelDep3_Geo.ifc

Add GeoScienceFeature with VoxelGrid and associated IfcRealVoxelData and IfcIntegerVoxelData.

AdamMackellar12d committed:
Update IFCTunnelDep3_Geo.ifc

Aggregate 'Site 01' to IfcProject.

AdamMackellar12d committed:
Update to Geo_BookB

Separate out Geo_BookA and GeoBookB.
Add GeoScienceModel/PHYSICALPROPERTYDISTIBUTIONMODEL into spatial structure under Site 01 and attach IfcGeoScienceFeature/PHYSICALPROPERTYDISTIBUTIONMODEL to it.

AdamMackellar12d committed:
Update IFCTunnelDep3_Geo_BookB.ifc

Change the relationship for IfcGeoScienceModel->IfcGeoScienceFeature from Spatial Structure to Decomposition.

AdamMackellar12d committed:
Update IFCTunnelDep3_Sys.ifc

Added in Pset_PipeSegmentTypeCommon for IfcPipeSegment entities.
